website/docs/r/key_vault_managed_hardware_security_module_key_rotation_policy.html.markdown

```diff
@@ -0,0 +1,66 @@
+---
+subcategory: "Key Vault"
+layout: "azurerm"
+page_title: "Azure Resource Manager: azurerm_key_vault_managed_hardware_security_module_key_rotation_policy"
+description: |-
+  Manages a Managed HSM Key rotation policy.
+---
+
+# azurerm_key_vault_managed_hardware_security_module_key_rotation_policy
+
+Manages a Managed HSM Key rotation policy.
+
+## Example Usage
+
+```hcl
+resource "azurerm_key_vault_managed_hardware_security_module_key" "example" {
+  name           = "example-key"
+  managed_hsm_id = azurerm_key_vault_managed_hardware_security_module.example.id
+  key_type       = "EC-HSM"
+  curve          = "P-521"
+  key_opts       = ["sign"]
+}
+
+resource "azurerm_key_vault_managed_hardware_security_module_key_rotation_policy" "example" {
+  managed_hsm_key_id = azurerm_key_vault_managed_hardware_security_module_key.example.id
+  expire_after       = "P60D"
+  time_before_expiry = "P30D"
+}
+```
+
+## Arguments Reference
+
+The following arguments are supported:
+
+* `expire_after` - (Required) Specify the expiration time of the key.
```

Comment:
is this a date? a duration? what format?
```suggestion
* `expire_after_date` - (Required) Specify the expiration time of the key.
```
```suggestion
* `expire_after_duration` - (Required) Specify the expiration time of the key.
```

Reply:
please detail what this actually means, is it related to the lifecycle actions/trigger not the key?